profiles/categories

```diff
@@ -1,4 +1,5 @@
 atom-package
+dev-lerna
```

Comment:
node-lerna

Reply:
Probabilmente mi è scappato quando ho cambiato dev-lerna in node-lerna